src/App.js

```diff
@@ -0,0 +1,90 @@
+// src/App.js
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import Home from './components/home';
+import Signup from './auth/signup';
+import Login from './auth/login';
+import { checkLoginStatus } from './redux/loginSlice';
+import Navbar from './components/navbar';
+import AddItem from './components/addItem';
+import ItemDetails from './components/itemDetails';
+import AddReservations from './components/addReservations';
+import DisplayReservations from './components/displayReservations';
+import DeleteItems from './components/deleteItems';
+import './stylesheets/App.css';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry (I found 13):

![image](https://github.com/geekelo/fenaj-schedula/assets/57421823/773f5dc9-92ab-417a-8cc4-b4a761d6e3f1)
  
That way, your code will meet the latest standards. 💪

Reply:
Thank you for this feedback.

Every attempt to modify the scripts according to your suggesting is throwing out an linter error as shown in the screenshot below. We are forced by linter to maintain our existing code structure.

Any suggestion on what to do?

@Meltrust 

<img width="735" alt="Screenshot 2023-12-14 at 11 40 41 AM" src="https://github.com/geekelo/fenaj-schedula/assets/7517876/41c7ef4c-47f9-4208-94c3-c2879f3a2586">

Reply:
@yemidada, this is a first.  Must be an update on the linter.   You can disregard this review comment.